src/main/java/com/deepoove/swagger/diff/model/ChangedParameter.java

```diff
@@ -27,7 +29,10 @@ public void setChangeRequired(boolean isChangeRequired) {
   }
 
   public boolean isChangeDescription() {
-    return isChangeDescription;
+    if (this.isChangeDescription) {
+      return true;
+    }
+    return leftParameter.getDescription() != null && rightParameter.getDescription() != null && !leftParameter.getDescription().equals(rightParameter.getDescription());
```

Comment:
Why have this evaluate every time it's called?

Reply:
When I was adding the functionality I think this field was not getting set in the right places, but I think I fixed that. Pushing the change up.